components/hacbs/jvm-build-service/base/kustomization.yaml

```diff
@@ -4,6 +4,26 @@ resources:
 patchesStrategicMerge:
 - self-binding.yaml
 
+patches:
+  - patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/env/-
+        value:
+          name: IMAGE_TAG
+          value: b911059a17e1efdbc2ddb496604e673d0c491c43
+    target:
+      kind: Deployment
+      name: hacbs-jvm-operator
+  - patch: |-
+      - op: add
+        path: /spec/template/spec/containers/0/env/-
+        value:
+          name: IMAGE_REPO
+          value: redhat-appstudio
```

Comment:
what is this IMAGE_REPO?

Reply:
yeah this one is not "required" but rather a "nice to have" wrt to the development flow that @stuartwdouglas created for us

in that development flow, `redhat-appstudio` as the repository for the image ref is changed to the developer's personal repository on `quay.io` (i.e. `gabemontero` for me).

so certainly the appstudio staging bot will not need to change this as jvm-build-service PRs merged.

but in my infra-deployment branch I'm using for development I've changed the value of this env var to `gabemontero`

Reply:
so this patch would be needed only in dev overlay and should be modified by preview.sh?

Reply:
ok I've pushed that update @Michkov PTAL

Reply:
So could you rebase to newest version and remove changes in apiexport?

Reply:
ok I've rebased this PR to latest infra-deployments main as of this comment
and I've updated the jvm-build-service commit sha to the last valid one from 7:52 AM US Eastern

that said, per the slack thread https://coreos.slack.com/archives/C03KN2B47GU/p1666189974098079 I pinged you on @Michkov something unfortunately seems wrong with my update https://github.com/gabemontero/jvm-build-service/blob/71d073569161dd3c1e4492f525cac748f84fc037/.tekton/controller-push.yaml and the controller on push pipelinerun is not getting generated on the staging cluster.  As a result, there is no controller quay image for commit 040c0513e26a05461e3be76db4835542dfa962de

I'm trying to reproduce somehow while I wait to hear from you.

But in the meantime we need to 

/hold

this PR until that problem is fixed.

Reply:
oh, on remove the apiexport changes ... not sure I follow.  I've changed what jvm-build-service permission claims need to be, so the apiexport changes here reflect that.

Reply:
false alarm on the controller push job ....PR 332 from jvm-build-service lead to https://console-openshift-console.apps.appstudio-stage.x99m.p1.openshiftapps.com/k8s/ns/jvm-build-service/tekton.dev~v1beta1~PipelineRun/jvm-build-controller-on-push-qzhhk succeeding 

/hold cancel

Reply:
also https://github.com/redhat-appstudio/infra-deployments/pull/842 is updated correctly

Reply:
and re apiexports I think I see now that https://github.com/redhat-appstudio/infra-deployments/pull/842/files is making the same edits as in this PR, since I made those updates in jvm-build-service

Reply:
apiexport changes reverted PTAL